src/memtable.rs

```diff
@@ -107,6 +107,7 @@ impl MemTable {
             );
             self.rewrite_count += rhs.rewrite_count;
             self.entry_indexes.append(&mut rhs.entry_indexes);
+            rhs.rewrite_count = 0;
```

Comment:
No need to change it.

Reply:
I see. Then I'm skeptical of why your change works. Because `Vec::append` will remove all entries in `rhs.entry_indexes`, that append count calculation inside drop incorrect.

Reply:
Yes. Memable drop occurs in two scenario: 1) at bootstrap, memtables merge with each other. I this case, The drop is useless because the GlobalStats is not shared. 2) `Command::clean`, which will remote target memable. In this case we need to do this clean up by Memable::drop

Reply:
Ok. How about adding a `clear` method to MemTable instead of doing it in drop?

Reply:
Actually, forget it. It's not a good idea.